src/validator_main.cpp

```diff
@@ -153,23 +154,26 @@ namespace
         std::sort(outs.begin(), outs.end());
         std::unique(outs.begin(), outs.end());
         if (initial_size != outs.size()) {
-            std::cerr << "Warning, duplicated outputs! will write just once to each output specified by -r/--report" << std::endl;
+            BOOST_LOG_TRIVIAL(warning) << "Duplicated outputs! will write just once to each output specified by -r/--report";
         }
 
         std::vector<std::unique_ptr<ebi::vcf::ReportWriter>> outputs;
 
         for (auto out : outs) {
-            if (out == ebi::vcf::DATABASE) {
+            if (out == ebi::vcf::DATABASE || out == ebi::vcf::TEXT) {
                 auto epoch = std::chrono::system_clock::now().time_since_epoch();
```

Comment:
if we don't extract this out of the `for` loop, files from the same run might get different timestamps